### ISSUE TITLE
How to fine-tune the existing Japanese model (recognition task)

### ISSUE THREAD

**Issue by HienDT27** [PaddlePaddle/PaddleOCR#1737]
Hi, I want to fine-turning the japanese recognition model with my dataset (created by text-renderer)
as keep the inference accuracy by inference model ( japan_mobile_v2.0_rec_infer)

- pretrained model was use : japan_mobile_v2.0_rec_train
- config file : rec_japan_lite_train.yml

I tried with 5000 training data images but the results not good .
can't recognition any character or sentence that not in training data.
(but use inference model ( japan_mobile_v2.0_rec_infer) can recognize it good )
55447904/104461679-fec98b80-55f2-11eb-9da6-033ad78e2bd7.jpg)

How can I do to improve the  fine-turning model
 (better than inference model ( japan_mobile_v2.0_rec_infer))

**Comment by littletomatodonkey** [comment]
The model we provided has seen many words and if you just finetune it one 5000 images, it may perform well on the words contained in 5000 images while not so well in others.
